apps/modernization-ui/src/apps/page-builder/page/management/edit/question/QuestionContent.tsx

```diff
@@ -110,6 +110,59 @@ export const QuestionContent = ({
                 </div>
             ) : (
                 <>
+                    <div className={styles.content}>
+                        <div className={styles.questionHeader}>
+                            <Heading level={2}>{name}</Heading>
+                            <div className={styles.identifier}>{`(${identifier})`}</div>
+                        </div>
+                        <div className={styles.questionContent}>
+                            <div className={styles.inputContent}>
+                                {(displayComponent === 1007 ||
+                                    displayComponent === 1013 ||
+                                    displayComponent === 1024 ||
+                                    displayComponent === 1025 ||
+                                    displayComponent === 1027 ||
+                                    displayComponent === 1028 ||
+                                    displayComponent === 1031) && (
+                                    <SelectInput
+                                        onChange={() => {}}
+                                        defaultValue={''}
+                                        options={conceptState}
+                                        data-testid="dropdown-input"
+                                    />
+                                )}
+                                {/* create custom checkbox component */}
+                                {/* need to create an api that grabs the race since it is in another table, once that is done a custom component can be created */}
+                                {displayComponent === 1001 && (
+                                    <SelectInput defaultValue={''} onChange={() => {}} options={conceptState} />
+                                )}
+
+                                {displayComponent === 1008 && (
```

Comment:
Issue: instead of duplicating all of this, can we narrow down what `isStandard` changes for the display and make the necessary check to change the individual piece?